src/Sylius/Bundle/WebBundle/Resources/config/routing/backend/product.yml

```diff
@@ -85,3 +85,19 @@ sylius_backend_product_show:
             repository:
                 method: findForDetailsPage
                 arguments: [$id]
+
+sylius_backend_product_enable:
+    path: /{id}/enable
```

Comment:
IMHO two routes are a good solution, using one you have to provide parameter and it'll be longer and more complicated overall.